model_compression_toolkit/qat/common/qat_get_quantizer_config.py

```diff
@@ -0,0 +1,106 @@
+
+
+def get_trainable_quantizer_weights_config(
+        n: common.BaseNode,
+        weights_quantization_candidates: List[TrainableQuantizerCandidateConfig] = None) -> TrainableQuantizerWeightsConfig:
+    """
+    Returns the relevant configurations for weights trainable quantizer
+
+    Args:
+        n: BaseNode
+
+    Returns:
+         TrainableQuantizerWeightsConfig: an object that contains the quantizer configuration
+    """
+    final_cfg = n.final_weights_quantization_cfg
+    return TrainableQuantizerWeightsConfig(final_cfg.weights_quantization_method,
+                                           final_cfg.weights_n_bits,
+                                           final_cfg.weights_quantization_params,
+                                           final_cfg.enable_weights_quantization,
+                                           final_cfg.weights_channels_axis,
+                                           final_cfg.weights_per_channel_threshold,
+                                           final_cfg.min_threshold,
+                                           weights_quantization_candidates)
+
+
+def get_trainable_quantizer_activation_config(
+        n: common.BaseNode,
+        activation_quantization_candidates: List[TrainableQuantizerCandidateConfig] = None) -> TrainableQuantizerActivationConfig:
+    """
+    Returns configurations for activation trainable quantizer
+
+    Args:
+        n: BaseNode
+
+    Returns:
+         TrainableQuantizerActivationConfig - an object that contains the quantizer configuration
+    """
+    final_cfg = n.final_activation_quantization_cfg
+    return TrainableQuantizerActivationConfig(final_cfg.activation_quantization_method,
+                                              final_cfg.activation_n_bits,
+                                              final_cfg.activation_quantization_params,
+                                              final_cfg.enable_activation_quantization,
+                                              final_cfg.min_threshold,
+                                              activation_quantization_candidates)
+
+
+def get_trainable_quantizer_quantization_candidates(n: common.BaseNode):
+    """
+    Returns quantization configuration candidates for activation and weights trainable quantizer
+
+    Args:
+        n: BaseNode
+
+    Returns:
+         weights_quantization_candidates - A list of configuration candidates for weights
+         activation_quantization_candidates - A list of configuration candidates for activation
+    """
+
+    # all candidates must have the same weights quantization method
+    weights_quantization_method = len(set([cfg.weights_quantization_cfg.weights_quantization_method for cfg in n.candidates_quantization_cfg]))
+    if weights_quantization_method > 1:
+        common.Logger.error(f'Unsupported candidates_quantization_cfg with different weights quantization methods')
```

Comment:
Add "pragma: no cover" to logger lines if you don't intend of adding tests that check the exception specifically.